client/components/home/hero/hero.js

```diff
@@ -1,21 +1,28 @@
+import { useEthers } from "@usedapp/core";
 import styles from "./hero.module.scss";
 import ThemedImage from "../../../lib/themedImage";
 
-const Hero = () => (
-  <main className={styles.hero}>
-    <h1>
-      Welcome to dAvatar
-    </h1>
-    <p>
-      Please connect with your Metamask to continue
-    </p>
-    <p className={styles.learnMore}>Learn more</p>
-    <ThemedImage
-      className={styles.arrow}
-      lightThemeSrc="/light-mode-icons/iconmonstr-arrow-65.svg"
-      darkThemeSrc="/dark-mode-icons/iconmonstr-arrow-65-white.svg"
-    />
-  </main>
-)
+const Hero = () => {
+  const { account } = useEthers();
+
+  const greeting = account ? <p>You are connected with the following account<br/><span className={styles.address}>{account}</span></p>: <p>"Please connect with your Metamask to continue"</p>;
```

Comment:
Ova kobaja od linije me potseti da nam treba prettier na ovom projektu.

Reply:
@carce fell free to add it :D Napravi task i dodaj slobodno